README.md

```diff
@@ -4,10 +4,41 @@ A 'package manager' for Glorious Eggroll's custom proton build.
 
 ## Installation
 
+There are currently a couple of ways to install `humble-lumpia`.
+
+**NOTE:** Installing `humble-lumpia` and its dependencies is not sufficient to 
+run the versions of `proton-ge-custom` that `humble-lumpia` will download. Be 
+sure to reference the [README](https://github.com/GloriousEggroll/proton-ge-custom/)
+in `proton-ge-custom` to determine what other dependencies your system requires.
+
+### AUR Package Installation
+
+If you are running **Arch Linux**, you can also download and install the 
+package maintained by [NINNiT](https://github.com/NINNiT) from the AUR. 
```

Comment:
What do you think about linking the word `AUR` to the actual package?

Reply:
I added a link as suggested in this latest batch of commits.